app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -37,11 +37,6 @@ public void attach() {
     }
 
     @Override
-    public void detach() {
```

Comment:
the problem comes when fragment is switched while loading.
For example 
In `AttendeesPresenter` - 
```
eventDataRepository.getAttendees(eventId, forceReload)
            .subscribe(attendees -> {
                ...
                attendeesView.showAttendees(attendees);
                ...
            ...
            });
```
Once the presenter makes this call, while the subscriber is still waiting for the result, if fragment is switched that is presenter is detached then view becomes null and on the result emitted it throws NPE for `attendeesView`. 

Detaching presenter is only good in activity's `onDestroy`.